add solution

77carlos71 opened the pull request:
[DEMO LINK](https://77carlos71.github.io/layout_landing-page/)

I had to create a new pull request because the previous one was completely broken

DanielCamposDevX reviewed (changes requested):
Disable page scrolling when menu is open.
Correct field validations on input to check all fields are required.
Mobile is broken.

77carlos71 commented:
I think I fixed what you asked for, but the mobile one was kinda broken — not sure if you meant the screen layout or the red background on the texts. If it was the red background, I fixed it.

pedro-ruas reviewed (changes requested):
Links are not with correct styling. All links on the page (header and beside the contact form) are using the default blue color. They should follow the design.
All fields in contact form should be required

77carlos71 commented:
I changed the link color from blue to white and updated the fields.

raulriato reviewed (changes requested):
Hey @77carlos71 this `pull request` is wrong. I've had to go to your repository to see that you have another `branch`, `develop_new` where it seems you have your code. You should transfer the code to the `branch` `develop`, because it's the one being reviewed. Otherwise we can't see your code.

Another tip: You should not change the `main.js` file. This task is about HTML and CSS, you should make it work by correcting your `index.html`, which you moved out from the `src`, for some reason. and the `style` files, never coding into the `main.js`.

Commit:
add solution

Commit:
add solution

77carlos71 commented:
I moved everything to the develop branch. I deleted all the code in the JS file, and the index.html was already outside the src folder when I cloned the project.

pedro-ruas reviewed (changes requested):
There are 3 issues to be solved now:
- There is horizontal scrolling in the page, and it should not
- The HTML code only exists in the branch `develop_new` as Raul mentioned. There is no HTML code in the index.html file in `develop`. But the styling only exists in `develop`. Checkout into `develop_new`, copy the HTML file content, checkout back into `develop` and paste the code replacing everything there
- All fields in the contact form should be `required` and they are not

Commit:
add solution

77carlos71 commented:
I think I’ve fixed everything: no more horizontal scroll, the HTML from develop_new is in develop, and all contact form fields are required

DanielCamposDevX reviewed (changes requested):
Mobile still Broken (maybe the deploy is still in the other branch)

Example:
<img width="611" height="869" alt="image" src="https://github.com/user-attachments/assets/882a5455-a394-477b-9b1c-2e58d5ce4f62" />

Commit:
Atualiza index.html com conteúdo da develop

Commit:
Atualiza index.html com conteúdo da develop

77carlos71 commented:
<img width="343" height="183" alt="Captura de tela 2025-10-28 184835" src="https://github.com/user-attachments/assets/9bedc4fc-8b89-4731-a2c6-1354bb77ab3d" />

<img width="1904" height="922" alt="Captura de tela 2025-10-28 184933" src="https://github.com/user-attachments/assets/cb4e5859-02c2-499f-aaca-35ad0e2e3a92" />

<img width="371" height="248" alt="Captura de tela 2025-10-28 185753" src="https://github.com/user-attachments/assets/56067c86-2b30-452a-aa6c-acd464e2d4c8" />

At 320px, which is what’s shown in Figma, it’s working. The only difference would be the browser, but it’s the same base code as Chrome

DanielCamposDevX reviewed (approved):
This is a educational pourpouse aprove. Just remember. responsivity is for all screens and sizes, not just for the figma sizes.
